docs/ldml/tr35.md

```diff
@@ -396,7 +396,7 @@ The different identifiers can be converted to one another as described in this s
 
 ##### <a name="Language_Tag_to_Locale_Identifier" href="#Language_Tag_to_Locale_Identifier">BCP 47 Language Tag to Unicode BCP 47 Locale Identifier</a>
 
-A valid [[BCP47](#BCP47)] language tag can be converted to a valid Unicode BCP 47 locale identifier according to [Annex C. LocaleId Canonicalization](#LocaleId_Canonicalization)
+A valid [[BCP47](#BCP47)] language tag can be converted to a valid Unicode BCP 47 locale identifier according to [Annex C. LocaleId Canonicalization](#LocaleId_Canonicalization).
```

Comment:
Is there a reason `BCP47` was not updated to `BCP 47` here to be consistent with spacing?